app/consolidator/scheduler/JobScheduler.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package consolidator.scheduler
+
+import akka.actor.{ ActorSystem, Props }
+import com.typesafe.akka.extension.quartz.QuartzSchedulerExtension
+import com.typesafe.config.ConfigRenderOptions
+import javax.inject.{ Inject, Singleton }
+import play.api.Configuration
+import play.api.libs.json.Json.parse
+
+import scala.collection.JavaConverters._
+
```

Comment:
Reads the jobs configured in application.conf and creates job schedules that periodically sends job params to the receiving actor (which executes the consolidation job)